README.md

```diff
@@ -1,25 +1,108 @@
 # Quarkus Backstage
 
-[![Version](https://img.shields.io/maven-central/v/io.quarkiverse.backstage/quarkus-backstage?logo=apache-maven&style=flat-square)](https://search.maven.org/artifact/io.quarkiverse.backstage/quarkus-backstage)
+[![Version](https://img.shields.io/maven-central/v/io.quarkiverse.backstage/quarkus-backstage?logo=apache-maven&style=flat-square)](https://central.sonatype.com/artifact/io.quarkiverse.backstage/quarkus-backstage-parent)
 
-## Welcome to Quarkiverse!
+Generate Backstage Catalog Information as part of the Quarkus build or the Quarkus CLI.
 
-Congratulations and thank you for creating a new Quarkus extension project in Quarkiverse!
+## Features
 
-Feel free to replace this content with the proper description of your new project and necessary instructions how to use and contribute to it.
+- Generate the catalog-info.yaml for the Quarkus application
+- Command Line interface to install / uninstall and Component to Backsatge
+- Orchestrate Quarkus extensions:
```

Comment:
What do you want to tell to the users using as word `orchestrate` ?

Reply:
That it coordinates, configures extensions so that they work in harmony with each other and with `quarkus-backstage`.

Reply:
`Orchestrate` => `Operate closely with the following Quarkus extensions` maybe 
We should here mention that Backstage should have been configured to with their corresponding plugin !